### ISSUE TITLE
Project status, new official fork?

### ISSUE THREAD

**Issue by bitsofinfo** [breser/git2consul#180]
Are any of the PRs or issues going to be addressed? If not is anyone out there interested in making a community fork where we can move this forward? Great tool

**Comment by boarder981** [comment]
Agreed.  Great tool, but has some serious flaws/bugs in my experience (see [this article](https://github.com/breser/git2consul/issues/177) for specifics)

**Comment by gshiva** [comment]
We are at the same stage, would like @HashiCorp to take over this one. We have some fixes and hopefully would be able to put it back here

**Comment by bitsofinfo** [comment]
https://github.com/breser/git2consul/issues/165#issuecomment-437561956

**Comment by ytjohn** [comment]
I have been looking as well.  I think the  most actively maintained fork is the one by [datadog](https://github.com/DataDog/git2consul). They actively maintain it for their own [environment](https://www.datadoghq.com/blog/engineering/consul-at-datadog/).  That said, I have no affiliation with them -- they might not want to take on processing community PRs.

I'm also looking closely at [Cimpress-MCP/go-git2consul](https://github.com/Cimpress-MCP/go-git2consul) - a port into go. The author warns it's not production ready yet, so I'll want to thoroughly test it  out in my environment.

**Comment by seanmalloy** [comment]
We have just released a fork of [Cimpress-MCP/go-git2consul](https://github.com/Cimpress-MCP/go-git2consul) today. Here is the fork with some additional features added [KohlsTechnology/git2consul-go](https://github.com/KohlsTechnology/git2consul-go).

We look forward to anyone in the community opening issues or pull requests.

**Comment by stevenscg** [comment]
Thanks @seanmalloy for the fork! I plan to try it out in our environments. We rely on the yaml expansion quite heavily.

If anybody else is looking for it, the fork point from the old project is noted in [ACKNOWLEDGEMENT.md ](https://github.com/KohlsTechnology/git2consul-go/blob/master/ACKNOWLEDGEMENT.md).
